src/actions/reject.ts

```diff
@@ -23,4 +24,34 @@ export async function reject(ignoreSslErrors?: boolean) {
 
     core.debug('Deployment strategy selected is Canary. Deleting baseline and canary workloads.');
     canaryDeploymentHelper.deleteCanaryDeployment(kubectl, TaskInputParameters.manifests, includeServices);
+}
+
+export async function rejectBlueGreen(ignoreSslErrors?: boolean) {
+    const kubectl = new Kubectl(await utils.getKubectl(), TaskInputParameters.namespace, ignoreSslErrors);
+
+    if (!blueGreenDeploymentHelper.isBlueGreenDeploymentStrategy()) {
+        core.debug('Strategy is not blue-green deployment. Invalid request.');
+        throw ('InvalidPromotetActionDeploymentStrategy');
+    }
+
+    let inputManifestFiles: string[] = getManifestFiles(TaskInputParameters.manifests);
+
+    if(blueGreenDeploymentHelper.isIngressRoute()) {
+        await blueGreenDeploymentHelper.blueGreenRejectIngress(kubectl, inputManifestFiles);
+    } else if (blueGreenDeploymentHelper.isSMIRoute()) {
+        await blueGreenDeploymentHelper.blueGreenRejectSMI(kubectl, inputManifestFiles);
+    } else {
+        core.debug('deleting new deployment');
+        await blueGreenDeploymentHelper.blueGreenReject(kubectl, inputManifestFiles);
+    }
+}
+
+function getManifestFiles(manifestFilePaths: string[]): string[] {
+    const files: string[] = utils.getManifestFiles(manifestFilePaths);
+
+    if (files == null || files.length === 0) {

```

Comment:
`if (!files)`

Reply:
![image](https://user-images.githubusercontent.com/25762922/86320591-d41a7580-bc54-11ea-9b7b-af5898ad05e4.png)

Reply:
Also, what's the point of this wrapper function? You could have added the throw logic in the utils function only right